<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtModelPanel</class>
 <widget class="QWidget" name="qtModelPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="treeviewLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="DomainButtonLayout">
     <item>
      <widget class="QPushButton" name="AddDomainset">
       <property name="text">
        <string>Add Domain Set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RemoveDomainset">
       <property name="text">
        <string>Remove Domain Set</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="BCButtonLayout">
     <item>
      <widget class="QPushButton" name="AddBC">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Add BC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RemoveBC">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Remove BC</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>202</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
